src/test/steps/deptCentralSteps.ts

```diff
@@ -0,0 +1,18 @@
+import { Given, Then, When } from '@cucumber/cucumber';
+import { pageFixture } from "../../core/pageFixture";
+import HomePage from '../../pages/homePage';
+
+let homePage:HomePage;
+
+Given('I go to {string}', {timeout:20000}, async function (website: string) {
+    await pageFixture.page.goto(website);
+    homePage = new HomePage(pageFixture.page);
+});
+
+Then('I can see that the title {string} is displayed and matches', {}, async function (title) {
+    await homePage.isVisibleTimeOffTitle();
```

Comment:
instead of coding two steps to verify two different texts, it'd be great if we can have one single step that covers both checks